server/utils/playlist.js

```diff
@@ -0,0 +1,322 @@
+const app = require('../server')
+const SpotifyWebApi = require('spotify-web-api-node');
+
+function getAccessToken(userID = null) {
+  const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.SPOTIFY_CLIENT_ID,
+    clientSecret: process.env.SPOTIFY_CLIENT_SECRET,
+    redirectUri: process.env.SITE_URL || 'http://localhost:8080',
```

Comment:
Loopback runs on port 3000 locally:
```suggestion
    redirectUri: process.env.SITE_URL || 'http://localhost:3000/auth',
```